charts/rancher-webhook/rancher-webhook/0.1.200-rc1+up0.1.2-rc1/Chart.yaml

```diff
@@ -3,7 +3,6 @@ annotations:
   catalog.cattle.io/hidden: "true"
   catalog.cattle.io/namespace: cattle-system
   catalog.cattle.io/os: linux
-  catalog.cattle.io/rancher-version: '>= 2.5.10 <2.5.99'
```

Comment:
We may need this in the future for airgap purposes, but we can remove until then.